html/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Homepage</title>
+
+
+</head>
+<body>
+    
+    <h1 align="center">My first Homepage</h1>
+
+<div align="center">
+    <figure> 
+      <img
+      src="https://tractive.com/blog/wp-content/uploads/2018/04/qualzucht-hund-die-auswirkungen-von-ueberzuechteten-hunden-ueberzuechtung-hund-500x450.jpg"
+      alt="a picture of a cute dog"/>
+      </figure>
+
+    <div id="nav" align="center">
+        <p>
+        <a href="#about-me"><strong>About Me</strong><a> &ensp;
+        <a href="#skills"><strong>Skillset</strong></a> &ensp;
+        <a href="#contact"><strong>Contact</strong></a>
+        </p>
+      </div>
+<br>
+      <main>
+        <div id="about-me">
+            <h2 align="center">About me</h2>
+            <p>
+             Lorem ipsum dolor sit amet consectetur adipisicing elit.
+             Reiciendis saepe illo atque facilis nemo! Aliquid assumenda 
+             iste neque, laudantium odio, excepturi accusamus inventore 
+             quia illo accusantium a ad soluta doloribus?
+            </p>
+        
```

Comment:
i think you're missing some closing tags for the **container**: `</div>`
otherwise i looks really good 👍

Reply:
oh, you´re right - thanks!! :) @gitboej